### PR TITLE
Create contribution.md

### PR DESCRIPTION
**Pull request by mapedraza**
Creating contribution.md to fill the requeriments of CII Best practices

### CODE REVIEW

File: docs/contribution.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+-   Code must be developed following the branching model and change log policies defined below.
+-   For any new feature added, unit tests must be provided, following the example of the ones already created.
+
+In order to start contributing:
+
+1.  Fork this repository clicking on the "Fork" button on the upper-right area of the page.
+2.  Clone your just forked repository:
+
+```bash
+git clone https://github.com/your-github-username/iotagent-json.git
```
Comment (fgalan) [inline docs/contribution.md:79]:
```suggestion
git clone https://github.com/your-github-username/sigfox-iotagent.git
```

File: docs/contribution.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+```
+
+3.  Add the main iotagent-json repository as a remote to your forked repository (use any name for your remote
+    repository, it does not have to be iotagent-json, although we will use it in the next steps):
+
+```bash
+git remote add iotagent-json https://github.com/telefonicaid/iotagent-json.git
```
Comment (fgalan) [inline docs/contribution.md:86]:
```suggestion
git remote add iotagent-json https://github.com/telefonicaid/sigfox-iotagent.git
```

File: docs/contribution.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+1.  Change to your local `master` branch (in case you are not in it already):
+
+```bash
+git checkout master
+```
+
+2.  Fetch the remote changes:
+
+```bash
+git fetch iotagent-json
```
Comment (fgalan) [inline docs/contribution.md:101]:
```suggestion
git fetch sigfox-iotagent
```

File: docs/contribution.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+In order to start contributing:
+
+1.  Fork this repository clicking on the "Fork" button on the upper-right area of the page.
+2.  Clone your just forked repository:
+
+```bash
+git clone https://github.com/your-github-username/iotagent-json.git
+```
+
+3.  Add the main iotagent-json repository as a remote to your forked repository (use any name for your remote
```
Comment (fgalan) [inline docs/contribution.md:82]:
```suggestion
3.  Add the main sigfox-iotagent repository as a remote to your forked repository (use any name for your remote
```

File: docs/contribution.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    repository, it does not have to be iotagent-json, although we will use it in the next steps):
+
+```bash
+git remote add iotagent-json https://github.com/telefonicaid/iotagent-json.git
+```
+
+Before starting contributing, remember to synchronize the `master` branch in your forked repository with the `master`
+branch in the main iotagent-json repository, by following this steps
```
Comment (fgalan) [inline docs/contribution.md:90]:
```suggestion
branch in the main sigfox-iotagent repository, by following this steps
```

File: docs/contribution.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+
+1.  Fork this repository clicking on the "Fork" button on the upper-right area of the page.
+2.  Clone your just forked repository:
+
+```bash
+git clone https://github.com/your-github-username/iotagent-json.git
+```
+
+3.  Add the main iotagent-json repository as a remote to your forked repository (use any name for your remote
+    repository, it does not have to be iotagent-json, although we will use it in the next steps):
```
Comment (fgalan) [inline docs/contribution.md:83]:
```suggestion
    repository, it does not have to be sigfox-iotagent, although we will use it in the next steps):
```

File: docs/contribution.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+If you don't see your idea listed, and you think it fits into the goals of this guide, do one of the following:
+
+-   **If your contribution is minor,** such as a typo fix, open a pull request.
+-   **If your contribution is major,** such as a new guide, start by opening an issue first. That way, other people can
+    weigh in on the discussion before you do any work.
+
+### Pull Request protocol
+
+As explained in ([FIWARE Development Guidelines](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Developer_Guidelines))
```
Comment (fgalan) [inline docs/contribution.md:31]: Broken link?

(Not sure if in other similar PRs we took the decision of using this link even broken and to be solved in the future)

Comment (mapedraza) [reply docs/contribution.md:31]: Similar in other PRs, we decided to use this link while finding the fixed link / another alternative

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**